src/App.js

```diff
@@ -59,13 +61,16 @@ const App = () => {
   const updateDeck = (deck, id) => {
     console.log("update invoked")
   }
+
   return (
     <>
       <Header currentUser={currentUser} logout={logout}/>
       <Routes>
         <Route path="/" element={<Home />} />
         <Route path="/decknew" element={<DeckNew createDeck={createDeck} currentUser={currentUser}/>} />
+        <Route path="/decks" element={<DeckIndex decks={decks}/>} />
         <Route path="/decks/:deck_id" element={<CardIndex decks={decks} cards={cards}/>} />
+        <Route path="mydecks/:deck_id/mycards/:card_id/edit" element={<CardEdit cards={cards} updateCard={updateCard} />} />
```

Comment:
I am just wondering if you need all of these params for this route?  Will card_id be unique enough to give you what you need without deck_id as a param as well.  You can navigate to the specific card from your deck show page, sending the card id you wish to update.